### DISCUSSION TITLE
Online version

### DISCUSSION THREAD

**Discussion by lnatal2** [adityaoberai/Alt-Text-Generator#33] [Q&A]
Online version is not working, with the following msg error: Error occured: An unknown error occured, please try again.

**Comment by adityaoberai** [comment]
Hi
I am currently in the middle of some travel and I hit the Azure spending limit unfortunately.

I'll set up a workaround and update here as soon as possible.

**Reply by lnatal2** [reply]
Thanks for your prompt reply.
